Makefile

```diff
@@ -39,24 +29,17 @@ spectral-security.yml: ./rules/  ./security/
 spectral-full.yml: spectral.yml spectral-security.yml
 	./rules/merge-yaml spectral.yml spectral-security.yml > $@
 
+# Build js bundle
 build: install rules
 	yarn build-js
 
+# Run test suite
 test-ui: install
 	yarn eslint
 	yarn test
-
 test:
 	bash test-ruleset.sh rules/ all
 	bash test-ruleset.sh security/ all
 
-# TODO remove and use https://www.npmjs.com/package/gh-pages
-gh-pages: rules build
-	rm dist -fr
-	git clone $(REPO_ORIGIN) $(TMPDIR) -b gh-pages
-	cp -r dist/* $(TMPDIR)
-	git -C $(TMPDIR) add .
-	git -C $(TMPDIR) -c user.name="gh-pages bot" -c user.email="gh-bot@example.it" \
-		commit -m "Script updating gh-pages from $(shell git rev-parse --short HEAD). [ci skip]"
-	git -C $(TMPDIR) push -q origin gh-pages
-	rm $(TMPDIR) -fr
\ No newline at end of file
+deploy: all
+	yarn deploy
```

Comment:
We can probably get rid of this altogether and just call the deploy script from CircleCI when we are ready, [like in the developers.italia.it repo](https://github.com/italia/developers.italia.it/blob/master/.circleci/config.yml#L88).

Reply:
Yes! So I can remove the deploy section in this PR and then when we are ready I can script a GitHub Action to manage deploy and test. (I would prefer GitHub Action over CircleCI if it is possible)

@ioggstream do you agree?

...or we can get rid of the entire MakeFile if we extract the spectral ruleset generation in a separate script... @ioggstream ?

Reply:
WRT:
- gh-actions for gh-pages: if that works, it is ok
- circleci: we use it for tests, but if `italia/` is moving to gh-action it is fine, as long as test runs 
- Makefile: I like to type `make test`. Unless it is a problem, I'd retain it.

Reply:
> gh-actions for gh-pages: if that works, it is ok
Yes it works, I have tried it on my forked [repo](https://emiliosp.github.io/api-oas-checker/)